pygdf/datetime.py

```diff
@@ -1,11 +1,26 @@
+import datetime as dt
+import time
+
 import numpy as np
-from . import columnops, _gdf
+import pandas as pd
+
+from . import columnops, _gdf, utils
 from .buffer import Buffer
-from .cudautils import compact_mask_bytes
 from libgdf_cffi import libgdf
 
 
+_unordered_impl = {
+    'eq': libgdf.gdf_eq_generic,
+    'ne': libgdf.gdf_ne_generic,
+}
+
+
+# class DatetimeColumn(numerical.NumericalColumn):
 class DatetimeColumn(columnops.TypedColumnBase):
+    # TODO - we only support nanoseconds (date64)
```

Comment:
Is this nanoseconds or milliseconds? Below we set the dtype as `datetime64[ms]`.